docs/replication-retry.md

```diff
@@ -59,24 +91,32 @@ See [Explanations](#explanations) for more detail.
     Request Body:
 
     ```sh
-    [{
-        bucket: <bucket>,
-        key: <key>,
-        versionId: <versionId>,
-        site: <site>
-    }]
+    {
+        Versions: [{
+            Bucket: <bucket>,
+            Key: <key>,
+            VersionId: <version-id>,
+            StorageClass: <site>,
+            Size: <size>,
```

Comment:
Size and LastModified fields should not be mandatory in the request, so better not document them (if the user provides the actual listing result from the GET it should work though, but a user may decide to create its own list from another data source as long as he complies with the API).

Also I guess in the description "This POST request retries a failed operation." we should say "retries a set of failed operations" instead.